docs/src/main/sphinx/admin/properties-memory-management.rst

```diff
@@ -56,6 +56,16 @@ system memory allocated by a query across all workers hits this limit it is
 killed. The value of ``query.max-total-memory`` must be greater than
 ``query.max-memory``.
 
+``query.max-total-memory-per-task``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-data-size`
+* **Default value:** none, and therefore unrestricted
+* **Session property:** ``query_max_total_memory_per_task``
+
+This is the max amount of user and system memory a task can use across the
+entire cluster. Support for using this property is experimental only.
```

Comment:
this is on node-level limit basis (task is node level entity); not cluster wide.

Reply:
Reworded.